proxy/round_tripper/proxy_round_tripper_test.go

```diff
@@ -1365,10 +1370,14 @@ var _ = Describe("ProxyRoundTripper", func() {
 								newCookies := resp.Cookies()
 								Expect(newCookies).To(HaveLen(2))
 								Expect(newCookies[0].Raw).To(Equal(sessionCookie.String()))
+
+								// This should fail when Golang introduces parsing for the Partitioned flag on cookies.
```

Comment:
Please also link https://github.com/golang/go/issues/62490 here.

Reply:
Good idea, thanks!
Done. I've added the link to the implementation and the test.